### PR TITLE
Vite and bento

### PR DESCRIPTION
**Pull request by GorlemZ**
I've added basic structure from vite and bento and some comment on the README

### CODE REVIEW

File: src/App.css
Comment (federico-ercoles) [inline src/App.css:1]: This file should not be needed, since all the style should come from the DS.

File: vite.config.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
```
Comment (federico-ercoles) [inline vite.config.ts]: A useful plugin for Typescript types' checking: [vite-plugin-checker](https://vite-plugin-checker.netlify.app/)

File: src/index.css
Comment (federico-ercoles) [inline src/index.css:1]: This file as well should not be needed.

File: src/vite-env.d.ts
Comment (federico-ercoles) [inline src/vite-env.d.ts:1]: I believe this file is actually needed for type checking, I would restore it.

File: vite.config.ts [hunk 2]
```diff
@@ -1,7 +1,14 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import checker from "vite-plugin-checker";
```
Comment (federico-ercoles) [inline vite.config.ts:3]: I don't see the plugin among the `devDependencies` in `package.json`, maybe you forgot to add that to the commit?

Comment (GorlemZ) [reply vite.config.ts:3]: yep I did some changes and forgot to re-add it from pnpm

File: src/vite-env.d.ts
Comment (GorlemZ) [reply src/vite-env.d.ts:1]: I think it is useful for custom types checks as environment variables (from what I've understood [here](https://stackoverflow.com/questions/67060543/how-to-recognize-env-variables-in-typescript-when-using-vite))
Maybe I'll need it later to store APIs keys, but I'm quite ok with deleting it for now. Is it ok?

Comment (federico-ercoles) [reply src/vite-env.d.ts:1]: It's also needed to resolve various kinds of imports (like images and media), I would keep it.

**Review by federico-ercoles — APPROVED** [review]
🚀

**Pull request merged**